### ISSUE TITLE
recognition Model predicts only whitespace

### ISSUE THREAD

**Issue by soufieneghribi** [PaddlePaddle/PaddleOCR#4876]
I am training a model to predict sentences.

I have 500K synth sentences 300K train and 200K evalutaion.

The accuracy were low (around 0.01) and after 27 epochs it started to increase (0.70) when I run evaluation I get 0% and when I test over samples I get only space

[2021/12/08 16:54:18] root INFO:         result:                0.8373171
How is this possible

latest evaluation: 

[2021/12/08 16:54:09] root INFO: cur metric, acc: 0.0, norm_edit_dis: 0.011587520081330238, fps: 5758.831746010819
[2021/12/08 16:54:09] root INFO: best metric, acc: 0.02206029447067296, norm_edit_dis: 0.7797898538511632, fps: 5771.436792803967, best_epoch: 28

[2021/12/08 16:54:14] root INFO: Architecture :
[2021/12/08 16:54:14] root INFO:     Backbone :
[2021/12/08 16:54:14] root INFO:         model_name : small
[2021/12/08 16:54:14] root INFO:         name : MobileNetV3
[2021/12/08 16:54:14] root INFO:         scale : 0.5
[2021/12/08 16:54:14] root INFO:         small_stride : [1, 2, 2, 2]
[2021/12/08 16:54:14] root INFO:     Head :
[2021/12/08 16:54:14] root INFO:         fc_decay : 1e-05
[2021/12/08 16:54:14] root INFO:         name : CTCHead
[2021/12/08 16:54:14] root INFO:     Neck :
[2021/12/08 16:54:14] root INFO:         encoder_type : rnn
[2021/12/08 16:54:14] root INFO:         hidden_size : 48
[2021/12/08 16:54:14] root INFO:         name : SequenceEncoder
[2021/12/08 16:54:14] root INFO:     Transform : None
[2021/12/08 16:54:14] root INFO:     algorithm : CRNN
[2021/12/08 16:54:14] root INFO:     model_type : rec
[2021/12/08 16:54:14] root INFO: Eval :
[2021/12/08 16:54:14] root INFO:     dataset :
[2021/12/08 16:54:14] root INFO:         data_dir : /elya1/ocr_sentences
[2021/12/08 16:54:14] root INFO:         label_file_list : ['/elya1/trdg/sent_arabic_val.txt']
[2021/12/08 16:54:14] root INFO:         name : SimpleDataSet
[2021/12/08 16:54:14] root INFO:         transforms :
[2021/12/08 16:54:14] root INFO:             DecodeImage :
[2021/12/08 16:54:14] root INFO:                 channel_first : False
[2021/12/08 16:54:14] root INFO:                 img_mode : BGR
[2021/12/08 16:54:14] root INFO:             CTCLabelEncode : None
[2021/12/08 16:54:14] root INFO:             RecResizeImg :
[2021/12/08 16:54:14] root INFO:                 image_shape : [3, 32, 320]
[2021/12/08 16:54:14] root INFO:             KeepKeys :
[2021/12/08 16:54:14] root INFO:                 keep_keys : ['image', 'label', 'length']
[2021/12/08 16:54:14] root INFO:     loader :
[2021/12/08 16:54:14] root INFO:         batch_size_per_card : 256
[2021/12/08 16:54:14] root INFO:         drop_last : False
[2021/12/08 16:54:14] root INFO:         num_workers : 8
[2021/12/08 16:54:14] root INFO:         shuffle : False
[2021/12/08 16:54:14] root INFO: Global :
[2021/12/08 16:54:14] root INFO:     cal_metric_during_train : True
[2021/12/08 16:54:14] root INFO:     character_dict_path : ppocr/utils/dict/arabic_dict.txt
[2021/12/08 16:54:14] root INFO:     character_type : arabic
[2021/12/08 16:54:14] root INFO:     checkpoints : None
[2021/12/08 16:54:14] root INFO:     debug : False
[2021/12/08 16:54:14] root INFO:     distributed : False
[2021/12/08 16:54:14] root INFO:     epoch_num : 120
[2021/12/08 16:54:14] root INFO:     eval_batch_step : [0, 2000]
[2021/12/08 16:54:14] root INFO:     infer_img : /elya1/ocr_sentences/KacstLetter_Medium_401.jpg
[2021/12/08 16:54:14] root INFO:     infer_mode : False
[2021/12/08 16:54:14] root INFO:     load_static_weights : False
[2021/12/08 16:54:14] root INFO:     log_smooth_window : 20
[2021/12/08 16:54:14] root INFO:     max_text_length : 45
[2021/12/08 16:54:14] root INFO:     pretrained_model : /home/sghribi/PaddleOCR/output/rec_arabic_lite_test_model/latest
[2021/12/08 16:54:14] root INFO:     print_batch_step : 10
[2021/12/08 16:54:14] root INFO:     save_epoch_step : 3
[2021/12/08 16:54:14] root INFO:     save_inference_dir : /home/sghribi/PaddleOCR/output/rec_arabic_lite_test
[2021/12/08 16:54:14] root INFO:     save_model_dir : /home/sghribi/PaddleOCR/output/rec_arabic_lite_test_model
[2021/12/08 16:54:14] root INFO:     use_gpu : True
[2021/12/08 16:54:14] root INFO:     use_space_char : False
[2021/12/08 16:54:14] root INFO:     use_visualdl : False
[2021/12/08 16:54:14] root INFO: Loss :
[2021/12/08 16:54:14] root INFO:     name : CTCLoss
[2021/12/08 16:54:14] root INFO: Metric :
[2021/12/08 16:54:14] root INFO:     main_indicator : acc
[2021/12/08 16:54:14] root INFO:     name : RecMetric
[2021/12/08 16:54:14] root INFO: Optimizer :
[2021/12/08 16:54:14] root INFO:     beta1 : 0.9
[2021/12/08 16:54:14] root INFO:     beta2 : 0.999
[2021/12/08 16:54:14] root INFO:     lr :
[2021/12/08 16:54:14] root INFO:         learning_rate : 0.001
[2021/12/08 16:54:14] root INFO:         name : Cosine
[2021/12/08 16:54:14] root INFO:     name : Adam
[2021/12/08 16:54:14] root INFO:     regularizer :
[2021/12/08 16:54:14] root INFO:         factor : 1e-05
[2021/12/08 16:54:14] root INFO:         name : L2
[2021/12/08 16:54:14] root INFO: PostProcess :
[2021/12/08 16:54:14] root INFO:     name : CTCLabelDecode
[2021/12/08 16:54:14] root INFO: Train :
[2021/12/08 16:54:14] root INFO:     dataset :
[2021/12/08 16:54:14] root INFO:         data_dir : /elya1/ocr_sentences
[2021/12/08 16:54:14] root INFO:         label_file_list : ['/elya1/trdg/sent_arabic_train.txt']
[2021/12/08 16:54:14] root INFO:         name : SimpleDataSet
[2021/12/08 16:54:14] root INFO:         transforms :
[2021/12/08 16:54:14] root INFO:             DecodeImage :
[2021/12/08 16:54:14] root INFO:                 channel_first : False
[2021/12/08 16:54:14] root INFO:                 img_mode : BGR
[2021/12/08 16:54:14] root INFO:             RecAug : None
[2021/12/08 16:54:14] root INFO:             CTCLabelEncode : None
[2021/12/08 16:54:14] root INFO:             RecResizeImg :
[2021/12/08 16:54:14] root INFO:                 image_shape : [3, 32, 320]
[2021/12/08 16:54:14] root INFO:             KeepKeys :
[2021/12/08 16:54:14] root INFO:                 keep_keys : ['image', 'label', 'length']
[2021/12/08 16:54:14] root INFO:     loader :
[2021/12/08 16:54:14] root INFO:         batch_size_per_card : 256
[2021/12/08 16:54:14] root INFO:         drop_last : True
[2021/12/08 16:54:14] root INFO:         num_workers : 8
[2021/12/08 16:54:14] root INFO:         shuffle : True
[2021/12/08 16:54:14] root INFO: train with paddle 2.2.0 and device CUDAPlace(0)
W1208 16:54:14.677000 1021395 device_context.cc:447] Please NOTE: device: 0, GPU Compute Capability: 7.5, Driver API Version: 11.4, Runtime API Version: 10.2
W1208 16:54:14.681891 1021395 device_context.cc:465] device: 0, cuDNN Version: 8.2.
[2021/12/08 16:54:17] root INFO: load pretrained model from ['/home/sghribi/PaddleOCR/output/rec_arabic_lite_test_model/latest']
[2021/12/08 16:54:17] root INFO: infer_img: /elya1/ocr_sentences/KacstLetter_Medium_401.jpg
[2021/12/08 16:54:18] root INFO:         result:                0.8373171
[2021/12/08 16:54:18] root INFO: success!

**Comment by soufieneghribi** [comment]
anyone?

**Comment by soufieneghribi** [comment]
有人可以回答我的问题吗？

**Comment by mehrdadazizi72** [comment]
@WenmuZhou 
Do you have any solution for this?
My accuracy for sentence-level recognition in persian is about 70% but have the same issue.

**Comment by mehdiimsv** [comment]
Has anyone solved this problem?  I have the same problem.
@soufieneghribi 
@mehrdadazizi72

**Comment by mehrdadazizi72** [comment]
@mehdiimsv unfortunately not. I abandoned my project.
